src/Drivers/CMakeLists.txt

```diff
@@ -1,8 +1,8 @@
 # Create examples and tests
-add_subdirectory(Dense)
+#add_subdirectory(Dense)
 if(HIOP_SPARSE)
   add_subdirectory(Sparse)
 endif()
-add_subdirectory(MDS)
-add_subdirectory(PriDec)
+#add_subdirectory(MDS)
+#add_subdirectory(PriDec)
```

Comment:
Should we include these directories again in the future?

Reply:
yes. My bad. I am running out of storage and hence I tried to remove some executable files.